### ISSUE TITLE
_to_tag_model_instances() missing 1 required positional argument: 'tag_kwargs'

### ISSUE THREAD

**Issue by realjumy** [coderedcorp/coderedcms#321]
Hi.

I'm having this error in 2 clean installations of CodeRedCMS:

```
TypeError at /admin/pages/3/edit/
_to_tag_model_instances() missing 1 required positional argument: 'tag_kwargs'
Request Method:	POST
Request URL:	https://mysite.com/admin/pages/3/edit/
Django Version:	3.0.6
Exception Type:	TypeError
Exception Value:	
_to_tag_model_instances() missing 1 required positional argument: 'tag_kwargs'
Exception Location:	/home/admin/web/mysite.com/venv/lib/python3.6/site-packages/modelcluster/contrib/taggit.py in add, line 44
Python Executable:	/home/admin/web/mysite.com/venv/bin/python3
Python Version:	3.6.9
Python Path:	
['/home/admin/web/mysite.com/cr_cms',
 '/home/admin/web/mysite.com/venv/bin',
 '/home/admin/web/mysite.com/venv/lib/python36.zip',
 '/home/admin/web/mysite.com/venv/lib/python3.6',
 '/home/admin/web/mysite.com/venv/lib/python3.6/lib-dynload',
 '/usr/lib/python3.6',
 '/home/admin/web/mysite.com/venv/lib/python3.6/site-packages']
Server time:	Fri, 22 May 2020 21:56:53 +0100
```

I installed CodeRedCMS yesterday and I got no problem whatsoever. Any ideas of what can be failing?

It only appears when I try to save content in any page.

**Comment by vsalvino** [comment]
I'm not sure, but I'm guessing there might be some pip package dependency down the line that updated and broke something. Can you run a `pip freeze` and paste the output?

**Comment by realjumy** [comment]
Thanks for the answer.

This is the output of `pip freeze`:

```
asgiref==3.2.7
beautifulsoup4==4.6.0
certifi==2020.4.5.1
chardet==3.0.4
click==7.1.2
coderedcms==0.18.1
decorator==4.4.2
Django==3.0.6
django-bootstrap4==1.1.1
django-eventtools==1.0.0
django-modelcluster==5.0.1
django-sendmail-backend==0.1.2
django-taggit==1.3.0
django-treebeard==4.3.1
djangorestframework==3.11.0
draftjs-exporter==2.1.7
future==0.18.2
geocoder==1.38.1
gunicorn==20.0.4
html5lib==1.0.1
icalendar==4.0.6
idna==2.9
l18n==2018.5
Pillow==6.2.2
pkg-resources==0.0.0
psycopg2-binary==2.8.5
python-dateutil==2.8.1
pytz==2020.1
ratelim==0.1.6
requests==2.23.0
six==1.15.0
sqlparse==0.3.1
Unidecode==1.1.1
urllib3==1.25.9
wagtail==2.8.2
wagtail-cache==1.0.0
wagtail-import-export==0.2
wagtailfontawesome==1.2.1
webencodings==0.5.1
Willow==1.3
```

**Comment by FlipperPA** [comment]
@realjumy Try `pip install django-taggit==1.2.0` for now. You can add that to your local project requirements.

Fixed in PR #322.

**Comment by realjumy** [comment]
Brilliant! thank you very much, @FlipperPA !
